client/components/LearnerViewDirective.js

```diff
@@ -239,6 +318,7 @@ tie.directive('learnerView', [function() {
           $scope.previousInstructions = [];
           $scope.nextButtonIsShown = false;
           $scope.feedbackParagraphs = introParagraphs;
+          $scope.unlockedQuestions[$scope.currentQuestionIndex] = true;
```

Comment:
Yep -- I'd go with `$scope.unlockedQuestions.push(false)` for the line in the for loop, but otherwise it looks good to me IMO.